### ISSUE TITLE
Estimate the 6DoF Object Pose

### ISSUE THREAD

**Issue by dingdingcai** [paulpanwang/POPE#3]
Hi, 

Thanks for the nice work!

I noticed that only the relative 3D rotation accuracy is evaluated and reported in the paper, how about the relative 3D translation? 
Is it possible to estimate the full 6DoF pose using POPE?

**Comment by YangHai-1218** [comment]
Hey, I have the same question.

**Comment by paulpanwang** [comment]
Indeed, you have a keen perception. In the scenario where we only have a single reference image, we can only estimate the scale based on the object's bounding box size in the 2D image (following the approaches of Gen6D and RelPose++). Training a scale regression network on a dataset that encompasses real-world scale relationships may be a viable solution.
